fuzzers/centipede/weak.c

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
```

Comment:
Not sure if I asked already but why is weak.c needed by centipede? It was just a hack I made for dataflow.

Reply:
Because we [enabled dataflow tracing](https://github.com/google/fuzzbench/blob/86f3b2447a88d2492b6922800f6ffd6018f66b07/fuzzers/centipede/fuzzer.py#L30) in `Centipede` as well : )
I recall it does work without this hack when tested locally earlier.

Reply:
Right I would be very surprised if Centipede needs this.

Reply:
Oh sorry, I just realised I mistyped: I meant to say it does *not* work without this hack.
But I will double-check tmr.

Reply:
You are right: we don't need `weak.c` here.
I did a bit more experiment today and can confirm that now.
Sorry about the confusion: I must have confused other error messages with it earlier.

I've fixed that locally and will push it tmr after rebasing to master.
Thanks for pointing this out!

Reply:
I remember why I added that `weak.c` now:
Benchmark `arrow_parquet-arrow-fuzz` needs it, otherwise it will fail because of undefined references to `__sanitizer_cov_store*` and `__sanitizer_cov_load*` functions.
See [this CI failure](https://github.com/google/fuzzbench/runs/7329782565?check_suite_focus=true).
Apparently, it is the only one needs it. I will ask Kostya.